index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile-Version</title>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <header class="header">
+      <a href="#"><h4>PEDRO MARQUES</h4></a>
+      <nav><img src="images/menu.svg" alt="Menu"></nav>
+    </header>
+    <section class="main">
+      <h1>Hey there. I’m Pedro <span> I’m a software developer</span></h1>
+      <p>
+         I can help you build a product , feature or website Look through
+         some of my work and experience! If you like what you see and 
+         have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <ul class="list-items">
+        <li><a href=""><img src="images/twitter.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/Linkedin-icon.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/medium.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/github.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/angellist.svg" alt="social-icons"></a></li>
+      </ul>
+        <ul>
+          <li><a href=""><img src="images/Disabled.svg" alt="disabled"></a></li>
+        </ul>
+          <h1>My Recent <br> Works</h1>
+        <ul>
+          <li><a href=""><img src="images/Indicator.svg" alt="indicator"></a></li>
+        </ul>
+    </section>
+    <div class="project">
```

Comment:
### OPTIONAL
- [ ] Kindly change this div tag to a section tag as this is the main container of the project, please add a `display:grid` instead of flex to the main container, this is part of the project requirements 👍🏿